### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,440 @@ Some tasks and individual language examples have labels as follows:
...
+            >
+              Python
+            </a>
+          </td>
+          <td align="middle">
+            <a
+              href="https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2641&repoName=osisoft%2Fsample-omf-temperature_sensor-python&branchName=master"
+            >
+              <img
+                src="https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OMF/osisoft.sample-omf-temperature_sensor-python?repoName=osisoft%2Fsample-omf-temperature_sensor-python&branchName=master"
```
Comment (kelseyduffy) [inline README.md:418]: should this image URL end with "&jobName=Tests_OCS"?

Comment (kelseyduffy) [inline README.md:418]: should this image URL end with "&jobName=Tests_OCS"

File: README.md [hunk 2]
```diff
@@ -11,14 +11,440 @@ Some tasks and individual language examples have labels as follows:
...
+                alt="Build Status"
+              />
+            </a>
+          </td>
+          <td align="middle">
+            <a
+              href="https://dev.azure.com/osieng/engineering/_build/latest?definitionId=2641&repoName=osisoft%2Fsample-omf-temperature_sensor-python&branchName=master"
+            >
+              <img
+                src="https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OMF/osisoft.sample-omf-temperature_sensor-python?repoName=osisoft%2Fsample-omf-temperature_sensor-python&branchName=master"
```
Comment (kelseyduffy) [inline README.md:428]: should this image URL end with "&jobName=Tests_EDS"?

File: README.md [hunk 3]
```diff
@@ -11,14 +11,440 @@ Some tasks and individual language examples have labels as follows:
...
```
Comment (kelseyduffy) [inline README.md:438]: should this image URL end with "&jobName=Tests_OnPrem"?

**Review by kelseyduffy — APPROVED** [review]
once the job names are added to the image url (if necessary), everything looks good to me

**Review by kelseyduffy — APPROVED** [review]
once the job names are added to the image url (if necessary), everything looks good to me

**Review by dcbroad3 — APPROVED** [review]

**Pull request merged**